File: src/BukkitPE/level/sound/GhastSound.php
<?php
namespace BukkitPE\level\sound;

use BukkitPE\math\Vector3;
use BukkitPE\network\protocol\LevelEventPacket;

class GhastSound extends GenericSound{
	public function __construct(Vector3 $pos, $pitch = 0){
		parent::__construct($pos, LevelEventPacket::EVENT_SOUND_GHAST, $pitch);
	}
}

File: src/main/java/net/BukkitPE/level/sound/GhastSound.java
package net.BukkitPE.level.sound;

import net.BukkitPE.math.Vector3;
import net.BukkitPE.network.protocol.LevelEventPacket;

/**
 * Created on 2015/11/21 by xtypr.
 * Package net.BukkitPE.level.sound in project BukkitPE .
 */
public class GhastSound extends GenericSound {
    public GhastSound(Vector3 pos) {
        this(pos, 0);
    }

    public GhastSound(Vector3 pos, float pitch) {
        super(pos, LevelEventPacket.EVENT_SOUND_GHAST, pitch);
    }
}
